packages/data-stores/src/plans/types.ts

```diff
@@ -61,7 +61,7 @@ export interface PricedAPIPlan {
 	product_slug: StorePlanSlug;
 	product_name_short: string;
 	product_type?: string;
-	bill_period: -1 | ( typeof PERIOD_LIST )[ number ];
+	bill_period: -1 | typeof PERIOD_LIST[ number ];
 
```

Comment:
Let's undo the changes in this file to fix the code style tests.

You can override the content of this file from trunk:
`git checkout origin/trunk -- packages/data-stores/src/plans/types.ts`